scripts/generate-tokens.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if ! command -v jq > /dev/null; then
+  printf "is jq installed?\n\n"
+  exit 1
+fi
+
+if ! command -v curl > /dev/null; then
+  printf "is curl installed?\n\n"
+  exit 1
+fi
+
+tokens_file=voorbeeld.tokens.json
+temp_dir=$(mktemp -d)
+trap "rm -rf ${temp_dir}" EXIT
+
+curl --silent -O --output-dir "${temp_dir}" "https://raw.githubusercontent.com/nl-design-system/themes/refs/heads/main/packages/voorbeeld-design-tokens/figma/${tokens_file}"
+
+for dir in packages/tokens/*-tokens; do
+  # strip prefix
+  component="${dir#packages/tokens/}"
+  # strip suffix
+  component="${component%-tokens}"
+  jq -f scripts/filter.jq --arg key "components/${component}/nl" "${temp_dir}/${tokens_file}" | jq -S . > "${dir}/tokens.json"
+done
```

Comment:
Ik stel voor dat we nog duidelijk moeten maken dat dit echt alleen een convenience is, maar dat de source of truth NIET in de themes repo zit, en dat elke wijziging aan de `tokens.json` files hier nauwkeurig en kritisch beoordeeld moeten worden.